docs/Getting Started/Example Apps/index.md

```diff
@@ -12,4 +13,208 @@ The example apps implement soft buttons, template text and images, a main menu a
+
+#### Bluetooth
+
+1. Make sure the build flavor ```multi_sec_offDebug ``` is selected.
+2. Ensure your phone is properly paired with the TDK
+3. Make sure Bluetooth is turned on - on Both the TDK and your phone
+4. Make sure apps are enabled on the TDK (in settings)
+!@
+
+@![javaEE, javaSE]
+### Introduction
+
+In this guide we take you through the steps to get our sample project, Hello Sdl, running and connected to Sdl Core as well as showing up on the generic HMI.
+
+Make sure that you follow the steps in [Installation](Getting Started With JavaEE/Installation) and [Integration Basics](Getting Started With JavaEE/Integration Basics) sections to create a new JavaEE SDL project before continuing this section.
+
+!!! NOTE
+The [Hello Sdl JavaEE](https://github.com/smartdevicelink/sdl_java_suite/tree/master/hello_sdl_java_ee) project includes samples for `SdlService` and `Main` classes that can be copied to your project.
+!!!
+
+### Getting Started
+
+We assume that you have [SDL Core](https://github.com/smartdevicelink/sdl_core) (We recommend Ubuntu 16.04) and an [HMI](https://github.com/smartdevicelink/generic_hmi) set up prior to this point. Most people getting started with this tutorial will be using Sdl Core and our Generic HMI. If you don't want to set up a virtual machine for testing, we offer [Manticore](https://smartdevicelink.com/resources/manticore/), which is a free service that allows you to test your apps in the cloud.
+
+!!! NOTE
+Sdl Core and an HMI or Manticore are needed to run Hello Sdl and to ensure that it connects.
+!!!
+
+### Configuring Core 
+
+To let Sdl Core connect to your app, first you will have to know the IP address of the machine that is running the Hello Sdl app. If you don't know what it is, running ```ifconfig``` in the terminal will usually let you see it for the interface you are connected with to your network. 
+
+After getting the IP address, you will have to set App ID, App Websocket Endpoint, and App Nicknames in Sdl Core to let it know where your instance of Hello Sdl is running. 
+
+!!! NOTE
+The App Websocket Endpoint contains the IP Address and port as the following: `ws://<ip address>:<port>/`.
+!!!
+
+
+#### Manticore
+If you are using Manticore, the app information can be easily set in the settings tab:
+
+![Main Screen](assets/manticore1.png)
+
+!!! NOTE
+Manticore needs to access you machine's IP address to be able to start a websocket connection with your cloud app. If you are hosting the cloud app on your local machine, you may need to do extra setup to make your machine publicly accessible. The other solution is to setup Core and HMI on your machine instead of using Manticore so Core can access your local IP address.
+!!!
+
+#### Sdl Core and Generic HMI
+If you are using Sdl Core and Generic HMI, you will have to add a policy table entry as the following one to the existing entries:
```

Comment:
```suggestion
If you are using SDL Core and Generic HMI, you will have to add a policy table entry for your app to the existing `app_policies` entries. The key for your policy table entry should be the same as your app ID.
```

Reply:
Needs to include the following:

* The name of policy table file: *sdl_preloaded_pt.json*
* The `storage` folder in the same directory as the *sdl_preloaded_pt.json* file needs to be deleted before changes made to the *sdl_preloaded_pt.json* take effect.

Reply:
I think I've done this.